users/models.py

```diff
@@ -242,6 +242,88 @@ def has_quiz(self) -> bool:
     def has_user_card(self) -> bool:
         return hasattr(self, "user_card")
 
+    @property
+    def quizs_answer_count(self) -> int:
+        answer_count = 0
+        for quiz in self.quizs.all():
```

Comment:
il y a moyen d'éviter les boucles for (et donc la démultiplication de requêtes SQL) avec un `annotate()`, mais ça peut venir dans un deuxième temps !

Reply:
Corrigé